src/compas_viewer/scene/sceneobject.py

```diff
@@ -128,6 +130,14 @@ def __init__(
 
         self._inited = False
 
+    def __eq__(self, other) -> bool:
+        if not isinstance(other, ViewerSceneObject):
+            return False
+        return self.settings == other.settings
+
+    def __hash__(self):
+        return hash(tuple(self.settings))
```

Comment:
converting to tuple will lost the value, you need to do something like this instead:
https://stackoverflow.com/questions/5884066/hashing-a-dictionary

Also if now you are not using the hash anyway you can just remove this.

Reply:
i'm using hash [here](https://github.com/compas-dev/compas_viewer/pull/193/files#diff-103b7cc7366c585a7b7a87153c827cc959d2a5abc2ce81740a10d5698fd23cb7R70). and i fix it so it can also catch the different of value